themes/default/layouts/page/leadership.html

```diff
@@ -0,0 +1,56 @@
+{{ define "hero" }}
```

Comment:
@gthuang 100%, i've reached out to get photo + social handles because I don't think she's on the website at all yet (which is where i got the needed info for others on this page) - could you help me get that data?

Reply:
can you use her LinkedIn photo? https://www.linkedin.com/in/elenitsa-staykova-1a56b02/

https://twitter.com/EliStaykova

Reply:
we touched base and i got what i need, thank you! will update on this PR - i'll retag you once i make all those changes, prob monday